_config/products.yml

```diff
@@ -140,9 +140,35 @@ devstudio:
   name: Red Hat CodeReady Studio
   url_path_fragment: devstudio
   streams:
-    simrel-2019-09:
-      12.x.Nightly:
+    simrel-2019-12:
```

Comment:
You also need to define simrel-2019-12 at the eclipse node level

Reply:
done.